Client/Frontend/Browser/Interstitial Pages/CertificateErrorPageHandler.swift

```diff
@@ -7,6 +7,10 @@ import Foundation
 import Shared
 import BraveShared
 
+extension CFNetworkErrors {
+  static let braveCertificatePinningFailed = CFNetworkErrors(rawValue: Int32.min)!
```

Comment:
is this safe? can't we use an enum's value `CFNetworkErrors.kXXXX` or something

Reply:
No because we don't know what is the exact range of ALL network errors. Sometimes WebKit can throw errors outside of this range :(
So we use an extreme number so it doesn't clash.